composer/lib/dependabot/composer/update_checker/version_resolver.rb

```diff
@@ -255,7 +255,8 @@ def handle_composer_errors(error)
               named_captures.fetch("url")
             raise Dependabot::GitDependenciesNotReachable, dependency_url
           elsif error.message.start_with?("Could not parse version") ||
-                error.message.include?("does not allow connections to http://")
+                error.message.include?("does not allow connections to http://") ||
+                error.message.match?("The `url` supplied for the path .* does not exist")
```

Comment:
Should this use `include?` to match the partial string? Assumed `match?` was for regexes

Reply:
Yeah it is, it should actually be `error.message.match?(/The `url` supplied for the path .* does not exist/)`, I guess it's being coerced into a regex but it's definitely not what I intended, cheers